framework/include/utils/MultiMooseEnum.h

```diff
@@ -99,25 +118,25 @@ class MultiMooseEnum : public MooseEnumBase
 
   ///@{
   /**
-   * Un-assign a value
+   * Un-assign, or unset a value
    * @param names - a string, set, or vector giving the name to erase from the enumeration values
    */
-  void erase(const std::string & names);
-  void erase(const std::vector<std::string> & names);
-  void erase(const std::set<std::string> & names);
+  void eraseSetValue(const std::string & names);
+  void eraseSetValue(const std::vector<std::string> & names);
+  void eraseSetValue(const std::set<std::string> & names);
   ///@}
 
   ///@{
   /**
    * Insert operators
    * Operator to insert (push_back) values into the enum. Existing values are preserved and
    * duplicates are stored.
-   * @param names - a string, set, or vector representing one of the enumeration values.
+   * @param names - a string, set, or vector representing the enumeration values to set.
    */
-  void push_back(const std::string & names);
-  void push_back(const std::vector<std::string> & names);
-  void push_back(const std::set<std::string> & names);
-  void push_back(const MultiMooseEnum & other_enum);
```

Comment:
the nice thing with push_back is that it lets you template on a vector OR a multiMooseEnum

Reply:
Do you think it's worth keeping the less clear name? I've just added it back, but deprecated it.

Reply:
I could see it being useful for templating. I have not needed it yet